ui/StatusQ/src/StatusQ/Core/Utils/FeatureFlags.qml

```diff
@@ -0,0 +1,7 @@
+pragma Singleton
+
+import QtQuick 2.14
+
+QtObject {
+    readonly property bool nimbusVerificationProxyEnabled: false
```

Comment:
It's generally not a good idea to store _any_ state inside a QML singleton; those are owned by the QML engine, are loaded and stay in memory for the whole application runtime.

Reply:
Thank you @caybro! I believe this matches @igor-sirotin comment to put the feature flag out on nim side correct? 
Please let me know if you have any feedbacks on the strategy as well?

Reply:
Yup, I'd just put it directly in NIM and expose it as a property to QML, just like it's done for other "experimental" features already

Reply:
>  ... just like it's done for other "experimental" features already

Wait, please tell me more about that? I for sure don't want to reinvent the wheel.